### ISSUE TITLE
Using mf_get_breaks() does not work with infinite values

### ISSUE THREAD

**Issue by JWGrade** [riatelab/mapsf#48]
If a variable with NAs is used in a chloro map, the method breaks = "quantiles" does not work. It seems that na.rm = TRUE is not set in the quantile-function:

```
Error in quantile.default(x = var, probs = seq(0, 1, 1/n), ...) : 
missing values and NaN's not allowed if 'na.rm' is FALSE
```

Other methods are doing fine (fisher, q6, etc.).

Besides this small issue, I love working with mapsf.

**Comment by rCarto** [comment]
Thank you, can you please provide a reproducible example?
I see:

``` r
library(mapsf)
m <- mf_get_mtq()
m[3:5, "MED"] <- NA
mf_map(
  x = m, 
  var = "MED", 
  type = "choro", 
  breaks = "quantile"
)
```

![](https://i.imgur.com/L02UX9o.png)

<sup>Created on 2022-11-30 with [reprex v2.0.2](https://reprex.tidyverse.org)</sup>

**Comment by JWGrade** [comment]
I have noticed that I have NaN and also Inf values in my data. I have used the mapsf function in a quarto document and here I get the error message above, which only refers to problems with NaN's. I didn't know about the inf values. I can reproduce the error with this example:

`library(mapsf)
m <- mf_get_mtq()
m[3:5, "MED"] <- Inf
mf_map(
  x = m, 
  var = "MED", 
  type = "choro", 
  breaks = "quantile"
)`

If I run the minimal example with inf values in a quarto document, I get the short (truncated) error message from above. If I run the example in the console or in an R script, I get the complete error message, which would have helped me directly:

`Error in quantile.default(x = var, probs = seq(0, 1, 1/n), ...) : 
  missing values and NaN's not allowed if 'na.rm' is FALSE
In addition: Warning message:
In classInt::classIntervals(var = x, n = nbreaks, style = breaks,  :
  var has infinite values, omitted in finding classes`

Result: My fault for not seeing the whole error message and producing inf values in my data....
